indico/htdocs/js/indico/jquery/categorypicker.js

```diff
@@ -0,0 +1,332 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function($) {
+    'use strict';
+
+    $.widget('indico.categorypicker', {
+        options: {
+            categoryId: 0,
+            height: null,
+            actionButtonText: $T.gettext("Select"),
+            openInDialog: false,
+            dialogTitle: $T.gettext("Select a category"),
+            onAction: function() {}
+        },
+
+        // Cache for data of already visited categories
+        _cache: {},
+
+        _typeaheadTemplate:
+            `<form>
```

Comment:
As much as I'd like this but nope :)

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals#Browser_compatibility